### PR TITLE
feat: Translation group email to group id

### PR DESCRIPTION
**Pull request by maciejmacq-dev**

### CODE REVIEW

File: locals.tf [hunk 1]
```diff
@@ -1,2 +1,4 @@
 locals {
+  groups_map      = { for g in data.google_cloud_identity_groups.groups.groups : g.group_key[0].id => g.name }
+  search_in_a_map = lookup(local.groups_map, var.group_email)
```
Comment (mariusz89016) [inline locals.tf:3]: what will happen when email is not in a `groups_map`?

File: variables.tf [hunk 2]
```diff
@@ -1,7 +1,7 @@
-# Example, compulsory input variable
-variable "group" {
+variable "group_email" {
   description = "Group ID"
   type        = string
+  default     = "example-group@google.com"
```
Comment (mariusz89016) [inline variables.tf:4]: I think we shouldn't provide default at all

**Review by mariusz89016 — COMMENTED** [review]
also, it's breaking change - we should bump major

File: variables.tf (hunk 2 above)
Comment (maciejmacq-dev) [reply variables.tf:4]: ok

File: locals.tf (hunk 1 above)
Comment (maciejmacq-dev) [reply locals.tf:3]: │ Error: Invalid function argument
│ 
│   on locals.tf line 3, in locals:
│    3:   search_in_a_map = lookup(local.groups_map, var.group_email)
│     ├────────────────
│     │ while calling lookup(inputMap, key, default...)
│     │ local.groups_map is object with 68 attributes
│ 
│ Invalid value for "inputMap" parameter: the given object has no attribute "d@g.pl".

Comment (mariusz89016) [reply locals.tf:3]: maybe we can handle that gracefully?

Comment (maciejmacq-dev) [reply locals.tf:3]: yes, probably we can :D

**Review by moss2k13 — APPROVED** [review]

**Pull request merged**